docs/cli-reference/clawback.md

```diff
@@ -116,7 +49,7 @@ Options:
 | Short Command | Long Command      | Type    | Required | Description                                                                                              |
 | :------------ | :---------------- | :------ | :------- | :------------------------------------------------------------------------------------------------------- |
 | -c            | --coin-id         | TEXT    | True     | The coin ID you want to claim                                                                            |
-| -d            | --fee             | INTEGER | False    | The fee in mojos for this transaction                                                                    |
+| -m            | --fee             | INTEGER | False    | The fee in XCH for this transaction                                                                      |
```

Comment:
Since we changed from mojo to XCH the type should be decimal (or float or whatever you use elsewhere for non-integer values)